pupil-spa/src/app/services/audit/auditEntry.ts

```diff
@@ -1,6 +1,6 @@
-export type AuditEntryType = 'WarmupIntroRendered' | 'WarmupCompleteRendered' | 'CheckComplete' |
-'CheckStarted' | 'QuestionRendered' | 'QuestionAnswered' | 'PauseRendered' | 'RefreshDetected' |
-  'UtteranceStarted' | 'UtteranceEnded';
+export type AuditEntryType = 'WarmupIntroRendered' | 'WarmupCompleteRendered' | 'CheckStartedAPICallSucceed' |
```

Comment:
CheckStartedAPICallSucceed is in the present tense, but the other verbs are all past tense.  Suggest rename to `CheckStartedAPICallSucceeded`